repository-data/application/src/main/resources/hcm-content/content/documents/administration/valuelists/publicationtopics.yaml

```diff
@@ -0,0 +1,81 @@
+/content/documents/administration/valuelists/publicationtopics:
```

Comment:
Could we `snake_case` the ValueList keys [`selection:key`] just to be inline with other ValueLists ?